.github/workflows/linux.yml

```diff
@@ -30,22 +30,24 @@ jobs:
         # We're not on a multi-user machine, so this is safe.
         run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           submodules: true
       - name: Install Go
         uses: actions/setup-go@v2
+        # TODO: with go-version-file and cache
+        # See https://github.com/actions/setup-go/issues/263
         with:
-          go-version: '1.18.1'
+          go-version: '1.18'
```

Comment:
This is better. Apparently, GitHub Actions treats "1.18" as "the latest version of 1.18.x", which is what we want.